content/docs/concepts/data_model.md

```diff
@@ -33,6 +33,9 @@ Note: The colons are reserved for user defined recording rules. They should not
  * Label values may contain any Unicode characters.
  * Labels with an empty label value are considered equivalent to labels that do not exist.
 
+NOTE: Beginning with Prometheus v3.0, the values of the `le` label of classic
+histograms and the `quantile` label in summaries are consistently normalized to
```

Comment:
are invariably? always?

Reply:
maybe link the migration guide? https://prometheus.io/docs/prometheus/latest/migration/#le-and-quantile-label-values

Reply:
Not sure if this is a good location for this factoid. The normalization is just an ingestion concern. Non-normalized `le` and `quantile` labels still fit the data model (and they even work with PromQL, it's just inconvenient if you have a mix of differently formatted `le` and `quantile` labels).